### PR TITLE
Release notes for v2.20.0

### PR DESCRIPTION
**Pull request by sanych-sun**

### CODE REVIEW

**Review by BorisDog — APPROVED** [review]

File: Release Notes/Release Notes v2.20.0.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# .NET Driver Version 2.20.0 Release Notes
+
+This is the general availability release for the 2.20.0 version of the driver.
+
+The main new features in 2.20.0 include:
+
+* Support for MongoDB server version 7.0.0 GA
+* Support for Queryable Encryption GA
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.20.0.md:8]: Only GA for equality, not range, queries.
- Support for equality query types in Queryable Encryption GA

File: Release Notes/Release Notes v2.20.0.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# .NET Driver Version 2.20.0 Release Notes
+
+This is the general availability release for the 2.20.0 version of the driver.
+
+The main new features in 2.20.0 include:
+
+* Support for MongoDB server version 7.0.0 GA
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.20.0.md:7]: Remove GA.

File: Release Notes/Release Notes v2.20.0.md (hunk 1 above)
Comment (sanych-sun) [reply Release Notes/Release Notes v2.20.0.md]: Done

File: Release Notes/Release Notes v2.20.0.md (hunk 2 above)
Comment (sanych-sun) [reply Release Notes/Release Notes v2.20.0.md]: Done

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**